packages/slate/src/changes/with-schema.js

```diff
@@ -67,42 +66,32 @@ function normalizeNodeAndChildren(change, node, schema) {
     normalizeNode(change, node, schema)
     return
   }
+  const normalizedKeys = []
+  let child = node.nodes.first()
+
+  let path = change.value.document.getPath(node.key)
```

Comment:
Can you fix these line breaks to match the rest of the code?

Reply:
Fixed. Thank you